docs/copilot/language-models.md

```diff
@@ -85,6 +101,21 @@ To update the provider details, such as the API key or endpoint URL:
 
 1. Update the provider details, such as the API key or endpoint URL.
 
+### Considerations
+
+There are a number of considerations when using your own language model API key in VS Code:
+
```

Comment:
I would add

"BYOK models only impact Chat, and do not control the model used for completions or other experiences like AI-powered commit message generation."